table-templates/template-table_13.qmd

```diff
@@ -0,0 +1,73 @@
+---
+title: "FDA Table 13"
+subtitle: "Patients With Common Adverse Events Occurring at >=XX% Frequency, Safety Population, Pooled Analyses"
+format: html
+---
+
+::: panel-tabset
+## Spec Screenshot
```

Comment:
Here again it is Spec instead of Spec.
Actually I don't have strong opinion on with/without the dot. But I think it should be the same in all pages.